frontend/build/server.mjs

```diff
@@ -46,8 +45,8 @@ const runServer = async () => {
 
       clientReq.pipe(proxy, { end: true });
     })
-    .listen(port);
-  console.log(`frontend running on http://${host}:${port}`);
+    .listen(process.env.FRONTEND_URL.split(":").slice(-1)[0]);
```

Comment:
Ah nice solution here 👍 
Worth noting there's a risk that the frontend url might not specify a port (e.g. if you're literally running on the default 443 and just want to use `https://mydomain.com` or whatever) but I suspect it's likely that in that case we won't be using this file to actually do any serving/hosting anyway, just building the files and then serving statically through some other medium